rotkehlchen/tests/unit/test_globaldb.py

```diff
@@ -793,6 +803,15 @@ def test_global_db_reset(globaldb):
     tokens_expected = cursor_clean_db.execute('SELECT COUNT(*) FROM assets;')
     tokens_local = cursor.execute('SELECT COUNT(*) FROM assets;')
     assert tokens_expected.fetchone()[0] + 3 == tokens_local.fetchone()[0]
+
+    # Check that historical price was not touched
```

Comment:
Can you also add some more details in the test to test that the other asset related tables have also been reset properly? You changed it to be properly deleted and replaced, but this test never failed before so it never caught it. Which means the test needs more deails